### ISSUE TITLE
Allow connecting input ports back to output ports

### ISSUE THREAD

**Issue by firstdorsal** [chrisjpatty/flume#170]
Currently it is only possible to connect nodes from a output port to the next input port.
I would love to connect them from the input of the second node to the output of the first node.
This of course makes no logic difference (and it should not) but would be awesome for the UX

**Comment by awhillas** [comment]
yeah, but then you'd get a infinite loop and your browser would lock up.

**Comment by firstdorsal** [comment]
> yeah, but then you'd get a infinite loop and your browser would lock up.

No, like i said this makes no logic difference, this is just a UX tweak.

I am not talking about connecting the output of the second node to the input of the first node (Yes that would create an infinite loop) but to connect the input of the second node to the output of the first node.
